flexmeasures/ui/templates/base.html

```diff
@@ -227,9 +227,16 @@
                     {% endfor %}
                 </ul>
             </li>
-            <div class="form-check form-switch" style="margin-left: auto;">
-                <input class="form-check-input" type="checkbox" {% if breadcrumb_info['current_asset_view'] == session.get("default_asset_view") %} checked {% endif %} id="defaultAssetView" onchange="setDefaultAssetView(this, '{{ breadcrumb_info["current_asset_view"] }}')">
-                <label class="form-check-label" for="defaultAssetView">Set as my default asset view</label>
+            <div class="row" style="margin-left: auto;">
+                <div class="col-auto form-check" title="Set this asset view as your default when visiting asset pages.">
+                    <input class="form-check-input" type="checkbox" {% if breadcrumb_info['current_asset_view'] == session.get("default_asset_view") %} checked {% endif %} id="defaultAssetView" onchange="setDefaultAssetView(this, '{{ breadcrumb_info["current_asset_view"] }}')">
+                    <label class="form-check-label" for="defaultAssetView">Set as default view</label>
+                </div>
+
+                <div class="col-auto form-check" title="This setting only affects graph pages with 7 or more sensors to show (with fewer sensors, the legends will be combined at the bottom anyway).">
```

Comment:
```suggestion
                <div class="col-auto form-check" title="If checked, legends for graphs will always be combined at the bottom (for the current user). This can be useful on narrower screens like tables). The default (off) means that with 7 or more sensors to show, the legends will be shown on the right, and per graph.">
```